src/functions/questionnaire/find-quiz/handler.ts

```diff
@@ -0,0 +1,60 @@
+import QuestionnaireService from "src/database/services/quiz-api-service";
+// import { CreateKeycloakApiService, type KeycloakApiService } from "src/apis/keycloak-api-service";
+import { middyfy } from "src/libs/lambda";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import type { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+
+const dynamoDb = new DocumentClient();
+const questionnaireService = new QuestionnaireService(dynamoDb);
+
+/**
+ * Lambda for finding quiz entry from DynamoDB.
+ *
+ * @param event event
+ * @returns quiz information as object
+ */
+
+const findQuizHandler: APIGatewayProxyHandler = async (
+  event: APIGatewayProxyEvent,
+) => {
+  const { id } = event.pathParameters || {};
+
+  try {
+    if (!id) {
+      return {
+        headers: {
+          "Content-Type": "application/json",
+        },
```

Comment:
```suggestion
```

Reply:
I'm quite sure you don't need to return headers here and you have not done this in the create function